app/src/main/index.js

```diff
@@ -119,6 +119,8 @@ function createWindow () {
     mainWindow.webContents.openDevTools()
   }
 
+  mainWindow.maximize()
```

Comment:
i thought you were opposed to this originally?

Reply:
This is temporary to make development faster on my end. I still like maximize for dev mode, but not for end-user mode.